tests/lib/cli.js

```diff
@@ -1112,4 +1112,11 @@ describe("cli", () => {
         });
     });
 
+    describe("when passing --info", () => {
+        it("should print out debugging information concerning the local environment", async() => {
+            await cli.execute("--info");
+            assert.isTrue(log.info.calledOnce);
```

Comment:
Should this assert that it's called twice since the second log has the information we're after?

Reply:
The very first `log.info()` is invoked before the `Promise` comes into play.